### DISCUSSION TITLE
Count of "^msgid" different from "All strings" in Weblate frontend

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#9431] [Q&A] [answered]
I do cloned the weblate repo of my project and then count the appearence of "msgid" in the beginning of each line of `de.po` file.

    cat de.po | grep "^msgid" | wc -l

This gives me `321`.

But the weblate frontend to show me 319 Strings. Where does the difference come from?

![image](https://github.com/WeblateOrg/weblate/assets/11861496/26c4bb87-f45a-40df-b400-43bb906e6f6c)

**Comment by nijel** [comment, marked as answer]
* The first string with empty msgid is PO file header.
* Plurals will match `^msgid` twice, better use `^msgid ` instead to avoid counting them twice.
